hash/msx1_cart.xml

```diff
@@ -18060,6 +18140,710 @@ legacy FM implementations cannot find it.
+		<year>2010</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="aorante/303bcn"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="psged - aorante-303bcn (2012) [v0.8b].rom" size="0x8000" crc="3be1ae30" sha1="23af545a914a061d9bae22542e480139b4a60ec9" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="psged04" cloneof="psged">
+		<description>PSGed (v0.4b)</description>
+		<year>2010</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="aorante/303bcn"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8001">
+				<rom name="psged - aorante-303bcn (2012) [v0.4b].rom" size="0x8001" crc="135b3416" sha1="f848ad905a22e177b80069f0497bb60ff4205485" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="puzzybox">
+		<description>Puzzybox</description>
+		<year>2021</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Gamecast Entertainment"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="slot" value="konami_scc"/>
+			<dataarea name="rom" size="0x10000">
+				<rom name="puzzybox - gamecast (2021).rom" size="0x10000" crc="454381e4" sha1="92d4601310eead197877835ff035a2ca6b10e1b6" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pwnd">
+		<description>PWND - Of je Worst Lust</description>
+		<year>2009</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Metal Soft"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="2"/>
+			<dataarea name="rom" size="0x4000">
+				<rom name="pwnd - of je worst lust - metal soft (2009).rom" size="0x4000" crc="69274ba7" sha1="681c30c7539ec738be2e36ddca408eed392f3b6d" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pwnda" cloneof="pwnd">
+		<description>PWND - Of je Worst Lust (alt)</description>
+		<year>2009</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Metal Soft"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="2"/>
+			<dataarea name="rom" size="0x4000">
+				<rom name="pwnd - of je worst lust - metal soft (2009) (alt).rom" size="0x4000" crc="51f201b3" sha1="2164c6ee1819f7fe763dbcc9ef366b62bf04aec6" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pwnd2">
+		<description>PWND Part 2 - Of Je Assembly Lust</description>
+		<year>2011</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="Metal Soft"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x2000">
+				<rom name="pwnd part 2 - of je assembly lust - metal soft (2011).rom" size="0x2000" crc="8e9f4013" sha1="6ac076f0c5dd09186fef2ebbee2b74221882538f" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
How are these titles supposed to be capitalised?  I’ve seen them as “PWND - Of je worst lust” and “PWND part 2 - Of je assembly lust” elsewhere.

Reply:
The 'Part' was wrong indeed, that make more sense as 'part'.
I used the more English way of capitalising the separate words. I've changed them back to the more commonly occurring 'Of je worst lust' and 'Of je assembly lust'.